promitor-agent-resource-discovery/values.yaml

```diff
@@ -155,10 +157,29 @@ rbac:
     name: promitor-resource-discovery
     annotations: {}
 
+    ## Set this to true if you plan on using Pod Security Policy
+    automountServiceAccountToken: false
+
 nodeSelector: {}
 
+## securityContext and containerSecurityContext are using secure defaults.
+## Only override if you have a good reason to.
 securityContext:
-  enabled: false
+  enabled: true
```

Comment:
Are we sure this does not break things?

Reply:
Pretty sure. Since the target port has been changed the pod doesn't need root privileges.